### DISCUSSION TITLE
Cannot connect to CouchDB instance

### DISCUSSION THREAD

**Discussion by fgrande** [Budibase/budibase#9359] [Help]
I have a problem with CouchDB, today we changed the scale of the project and now we have the following error from frontend:

{"message":"Cannot connect to CouchDB instance","status":500}

Before it was running fine... what can we check to solve the issue ? 

Regards
FabioG

PS - The installation was done through helm on GKE

**Comment by fgrande** [comment]
Hi, 

@shogunpurple I would better detail this issue, since I've made some more test.

Step 1 - Deploy of BB through helm, everything works fine
Step 2 - Change CouchDB definition, changing the clusterSize (from 3 to 2) and downsizing cpus/memory 
Step 3 - Apply changes with helm upgrade

BB is broken, it cannot access to CouchDB. Ok, maybe I opted for (too) small resources

Go back with previous cpus/memory sizes => BB is broken, there's no way to access with the previously created admin user.

Log errors are saying that 👍 
{database_does_not_exist,[{mem3_shards,load_shards_from_db,"_users",[{file,"src/mem3_shards.erl"},{line,400}]},{mem3_shards,load_shards_from_disk,1,[{file,"src/mem3_shards.erl"},{line,375}]},{mem3_shards,load_shards_from_disk,2,[{file,"src/mem3_shards.erl"},{line,404}]},{mem3_shards,for_docid,3,[{file,"src/mem3_shards.erl"},{line,97}]},{fabric_doc_open,go,3,[{file,"src/fabric_doc_open.erl"},{line,39}]},{chttpd_auth_cache,ensure_auth_ddoc_exists,2,[{file,"src/chttpd_auth_cache.erl"},{line,198}]},{chttpd_auth_cache,listen_for_changes,1,[{file,"src/chttpd_auth_cache.erl"},{line,145}]}]}

Regards
FabioG

**Reply by shogunpurple** [reply]
Hi @fgrande - can you show me the exact changes you applied with the `values.yaml`?

**Reply by fgrande** [reply]
I did this (indentation is on 2 chars): 

couchdb:
  clusterSize: 2
  resources:
    limits:
      cpu: 500m
      memory: 256Mi
    requests:
      cpu: 500m
      memory: 256Mi

Then it starts giving me error 500, so I reverted the config:

couchdb:
  clusterSize: 3
  resources:
    limits:
      cpu: 750m
      memory: 1000Mi
    requests:
      cpu: 7500m
      memory: 1000Mi

Now services are OK, but I cannot access with admin user.

**Reply by waza-ari** [reply]
@fgrande did you ever figure out what was going on here? I'm facing a similar issue
